roles/debian/hypervisor/tasks/main.yml

```diff
@@ -272,3 +272,29 @@
     name: ptp_vsock.service
     enabled: yes
     state: started
+
+- block:
```

Comment:
This should not be in the debian specific part. Could you move it in `playbooks/cluster_setup_network.yaml` ?

Reply:
for the CI to pass this PR, the package needs to be installed. But if the package is installed, then the cukinia-tests will fail for other PR (the package is not supposed to be there).
The workaround is to have the package installed by apt for the time being.
Once the PR is merged, we can add the package to build_debian_iso, change the cukinia-test, and then we can remove this part altogether.